Tests/SwiftParserTest/AvailabilityTests.swift

```diff
@@ -142,19 +142,21 @@ final class AvailabilityTests: XCTestCase {
       )
     )
 
+    assertParse(
+      """
+      @available(OSX 10.0.1, *)
+      func test() {}
+      """
+    )
+
     assertParse(
       """
       @available(OSX 1️⃣10e10)
       func test() {}
       """,
       diagnostics: [
-        DiagnosticSpec(message: "expected version tuple in version restriction", fixIts: ["insert version tuple"]),
-        DiagnosticSpec(message: "unexpected code '10e10' in attribute"),
-      ],
-      fixedSource: """
-        @available(OSX <#integer literal#>10e10)
-        func test() {}
-        """
+        DiagnosticSpec(message: "cannot parse version 10e10")
```

Comment:
Ideally the error message here would be `cannot parse version component '10e10'`, i.e. add *component* and wrap 10e10 in single quotes. You should get the quote wrapping automatically if you use `shortSingleLineContentDescription`.